api/collections.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const QuoteCollection = require('../models/quoteCollection.js');
+const router = require('express').Router();
+const HTTPStatus = require('http-status');
+
+router.get('/collections', listQuoteCollections);
+router.post('/collections', createQuoteCollection);
+router.get('/collections/:id', getQuoteCollection);
+router.put('/collections/:id', updateQuoteCollection);
+router.delete('/collections/:id', deleteQuoteCollection);
+router.post('/collections/:id/collaborators', addNewCollaborator);
+router.delete('/collections/:id/collaborators', removeCollaborators);
+router.get('/collections/:id/quotes', listCollectionQuotes);
+router.post('/collections/:id/quotes', addNewQuote);
+router.delete('/collections/:id/quotes', deleteQuote);
+
+module.exports = router;
+
+function listQuoteCollections(req, res, next) {
+  QuoteCollection.find(req.query)
+    .then(quoteCollection => res.status(HTTPStatus.OK).send(quoteCollection))
+    .catch(err => next(err));
+}
+
+function getQuoteCollection(req, res, next) {
+  QuoteCollection.findById(req.params.id)
+    .then(quoteCollection => res.status(HTTPStatus.OK).send(quoteCollection))
+    .catch(err => next(err));
+}
+
+function createQuoteCollection(req, res, next){
+  let collection = Object.assign({}, req.body, { owner: req.query.owner });
+  QuoteCollection.create(collection)
+    .then(quoteCollection => res.status(HTTPStatus.OK).send(quoteCollection))
+    .catch(err => next(err));
+}
+
+function deleteQuoteCollection(req, res, next){
+  QuoteCollection.findByIdAndRemove(req.params.id)
+    .then(() => res.status(HTTPStatus.NO_CONTENT).end())
+    .catch(err => next(err));
+}
+
+function updateQuoteCollection(req, res, next) {
+  let update = {
+    title: req.body.title,
+    description: req.body.description,
+    category: req.body.category
+  };
```

Comment:
`let update` is redundant here if you ask me. You can just send req.body to db query. Also that gives you ability to expand this route without code changes (e.g. if you want also to update something else later).
Later, when you add route validation you can make sure you are not receiving incorrect data or extra garbage.
@vladimyr second opinion? If he agrees look for similar occurrences throughout code base.